kodilla-stream/src/main/java/com/kodilla/stream/array/ArrayOperations.java

```diff
@@ -5,15 +5,19 @@
 public interface ArrayOperations {
 
     public static double getAverage(int[] numbers) {
-        IntStream.range(0, numbers.length)
-                .map(n -> numbers[n])
-                .forEach(System.out::println);
+        if (numbers == null) {
+            return 0;
+        } else if (numbers.length > 0) {
+            IntStream.range(0, numbers.length)
+                    .map(n -> numbers[n])
+                    .forEach(System.out::println);
 
-        double avg = IntStream.range(0, numbers.length)
-                .map(n -> numbers[n])
-                .average()
-                .getAsDouble();
-
-        return avg;
+            return IntStream.range(0, numbers.length)
+                    .map(n -> numbers[n])
+                    .average()
+                    .getAsDouble();
```

Comment:
Tu używasz konsturktu zwanego `Optional` i robisz `get` bez upewnienia się, czy on coś zawiera (IntelliJ powinein podświetlić ostrzeżenie).
Lepiej jest zrobić: `IntStream.range(0, numbers.length).map(n -> numbers[n]).average().orElse(0)`

Reply:
Właśnie nie byłem do końca pewien jak to zastosować. Intellij podawał na żółto tylko ostrzeżenie, że nie użyłem isPresent() metody ;)